openfe/setup/atom_mapping/perses_mapper.py

```diff
@@ -0,0 +1,85 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+
+"""
+The MCS class from Perses shamelessly wrapped and used here to match our API.
+
+"""
+
+from openmm import unit
+from openfe.utils import requires_package
+
+try:
+    from perses.rjmc.atom_mapping import AtomMapper, InvalidMappingException
+except ImportError:
+    pass    # Don't throw  error, will happen later
```

Comment:
We'll need a test that blocks the perses import and checks that the `requires_package` works.